test/unit/spec/MfaVerify_spec.js

```diff
@@ -1110,6 +1110,40 @@ function (Okta,
             });
           });
       });
+      it('on WARNING_TIMEOUT, after send/re-send, shows warning message for SMS after 30s idle', function () {
+        Util.speedUpPolling();
+        return setupFn().then(function (test) {
+          spyOn(LoginUtil, 'callAfterTimeout').and.callFake(function () {
+            // reducing the timeout to 0 so that test is fast.
+            return setTimeout(arguments[0], 0);
```

Comment:
PasscodeForm doesn't use `setTimeout` anymore right? So we need to change this test and remove all the references to `setTimeout`

Reply:
fixed